### ISSUE TITLE
Some ads not blocked

### ISSUE THREAD

**Issue by JoeTvx** [bromite/bromite#175]
https://googleads.g.doubleclick.net/ is not blocked on
https://slickdeals.net/ in version 70.0.3538.67 or newer. This used to be blocked earlier on in versions 63 and older. Not sure exactly where the stopped working.

Either in the code or in the app, if you could point me to the list of sites being blocked, I could dig in deeper. Thank you.

**Comment by DecodeMehh** [comment]
i see no ad.
can you post some screenshots?

**Comment by JoeTvx** [comment]
Halfway down the page and at the bottom of the page, I see two ads:

https://slickdeals.net/f/12198832-ebay-5-off-10-on-ebay-starting-tomorow-nov-5th-w-coupon-code-perkfive-deal-runs-between-nov-5th-8-00-am-pst-nov-6th-11-59-pm-pst-r

Screenshot in a minute...

**Comment by DecodeMehh** [comment]
Waiting for the screenshot.

I still see no ad on the page, make sure to update to latest 70.0.3538.87 version of bromite and check if AdBlock is enabled.

**Comment by JoeTvx** [comment]
70.0.3538.87 has the same issue. Adblock is enabled in the 3 dot menu. Adblock works fine on a few other sites.

BTW, since that page doesn't always serve that same ad, it takes a couple refreshes to see it.

**Comment by DecodeMehh** [comment]
Refreshed ~20 times and i still see no ad.

Keep in mind that the adblocker use uBlock filter and it is not bulletproof.

Wait for @csagan5 to investigate the issue.

**Comment by csagan5** [comment]
@JoeTvx you did not use the issue template, I am sorry but that is required.

Please re-open with the issue template, no screenshots needed. What is needed is the URLs of the ads which are not blocked.

You can get them by using remote devices in a Chrom* browser from desktop, see explanation here https://developers.google.com/web/tools/chrome-devtools/remote-debugging/

**Comment by JoeTvx** [comment]
Sure, where can I find the issue template? I don't see one in the readme or on the main website or the wiki. Thanks.

**Comment by csagan5** [comment]
@JoeTvx when you click "New issue" it shows a choice between bug and feature request:
![image](https://user-images.githubusercontent.com/32685696/48048574-3ed38f00-e19c-11e8-888d-cecca31155bc.png)

When you click the first, the text area contains the issue template already. You are expected to follow the template sections.

**Comment by csagan5** [comment]
> if you could point me to the list of sites being blocked, I could dig in deeper.

The ad lists being used are described here: https://github.com/bromite/bromite/blob/master/README.md#credits That is an exhaustive list.
The parsed adblock entries are here: https://github.com/bromite/bromite/blob/master/filters/adblock_entries.h

**Comment by csagan5** [comment]
It has been reported to me by another user that issue templates do not appear when submitting issues via mobile; I was always assuming that the issue template was being ignored, sorry for that.

I am working on a solution for this.

**Comment by JoeTvx** [comment]
Ah ok, indeed, I've been accessing GitHub via a mobile all this while. I'll check out the files he pointed me to and then repost through a desktop with the template soon.

**Comment by KevinSJ** [comment]
I was seeing the same issue with the latest version, so I used adguard to log dns requests and look for domain. I noticed a sdad.guru domain, which according to reddit user, belongs to a company called Instart Logic that 

>  "offers an ad-blocker circumvention service that replaces advertisement URLs with different URLs so that ad blockers cannot filter them"

More information [here](https://www.reddit.com/r/pihole/comments/76a6pn/instart_logic_offers_an_adblocker_circumvention/).

**Comment by JoeTvx** [comment]
@KevinSJ, would you mind filing a bug report on what you found. I can't tell if that's the same issue as I'm seeing, and I'm not sure I understand why older versions of Bromite would block the circumvention service. However, I'm currently unable to debug with a desktop that is required, so it would be great if you could file a bug report instead.

**Comment by csagan5** [comment]
Just to be clear: what is needed are the exact URLs of the ads, then with those further troubleshooting can be done. The remote debugger allows you to see that.

**Comment by nampud** [comment]
AFAIK Instart Logic is intentionally designed so that ad URLs and actual content URLs are indistinguishable by an URL based adblocker. uBlock Origin has a [companion script](https://github.com/gorhill/uBO-Extra/blob/master/contentscript.js) for blocking Instart, I think it could easily be adapted into this project.

**Comment by csagan5** [comment]
> AFAIK Instart Logic is intentionally designed so that ad URLs and actual content URLs are indistinguishable by an URL based adblocker.

The first goal of any bug report is reproducibility; it has not been achieved here so there is only speculation on what the ads escaping the blockade are.
